SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+We aim to keep Timescale safe for everyone. 
+Publicly disclosing bugs in a public forum puts everyone in the Timescale community at risk.
+
+## Supported Versions
+
+Our users should always upgrade to the latest version.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.6.x   | :white_check_mark: |
+| 1.x   | :x:                |
+
+
+## Reporting a Vulnerability
+
+If you find a vulnerability in our software the e-mail address to use to contact the Timescale Security Team is security@timescale.com.
```

Comment:
Some thoughts about the email message:
- maybe we can add some hints what information we would like to see in the email (type of the vulnerabilty, possible impact, repro steps)
- we can provide a PGP key to encrypt the mail
- response SLA